test/functional/wallet.py

```diff
@@ -57,8 +57,12 @@ def run_test (self):
         assert_equal(len(self.nodes[2].listunspent()), 0)
 
         # Send 21 BTC from 0 to 2 using sendtoaddress call.
+        # Locked memory should use at least 32 bytes to sign each transaction
```

Comment:
OK, but where is it guaranteed that [locked][used] is not re-used or freed after use? You test some other internal implementation behaviour here, not `getmemoryinfo`.